tests/AppBundle/Form/DataTransformer/TagArrayToStringTransformerTest.php

```diff
@@ -0,0 +1,124 @@
+<?php
+
+namespace Tests\AppBundle\Form\DataTransformer;
+
+use AppBundle\Entity\Tag;
+use AppBundle\Form\DataTransformer\TagArrayToStringTransformer;
+use Doctrine\Common\Persistence\ObjectManager;
+use Doctrine\ORM\EntityRepository;
+
+/**
+ * Tests that tags are transformed correctly using the data transformer.
+ *
+ * See http://symfony.com/doc/current/testing/database.html
+ *
+ * @author Jonathan Boyer <contact@grafikart.fr>
```

Comment:
AFAIK it's not common in tests classes, instead you can move it to `TagArrayToStringTransformer` class ;)